Explorer++/Explorer++/TabHandler.cpp

```diff
@@ -95,7 +95,8 @@ HRESULT Explorerplusplus::OnNewTab()
 		if (WI_IsFlagSet(fileFindData.dwFileAttributes, FILE_ATTRIBUTE_DIRECTORY))
 		{
 			auto pidl = selectedTab.GetShellBrowser()->GetItemCompleteIdl(selectionIndex);
-			return m_tabContainer->CreateNewTab(pidl.get(), TabSettings(_selected = true));
+			FolderColumns cols = selectedTab.GetShellBrowser()->ExportAllColumns();
+			return m_tabContainer->CreateNewTab(pidl.get(), TabSettings(_selected = true), nullptr, cols);

```

Comment:
One small style issue: this line is longer than 100 characters. I've been using clang-format to enforce a consistent set of style conventions across the codebase, one of which is that lines are at most 100 characters.
Could you run clang-format on this line? There's a .clang-format file in the repository root and I believe Visual Studio supports clang-format without needing to install anything extra, so it should be simple. Once the line has been selected in Visual Studio, select Edit > Advanced > Format Selection.

Reply:
Updated the PR